source/includes/api/_dashboard.md

```diff
@@ -350,10 +350,17 @@ curl -X DELETE https://api.resourcewatch.org/v1/dashboard/<id of the dashboard>
 
 ## Clone dashboard
 
-Clones an existing dashboard using its ID.
-If the original dashboard contains functioning widgets, they will be duplicated and the new ids will be used by the new dashboard.
+Clones an existing dashboard using its ID. If the original dashboard contains functioning widgets, they will be duplicated and the new ids will be used by the new dashboard. Data can be provided in the body of the request in order to override the data of the original dashboard. In the example on the side, the `name` of the dashboard will be overridden.
+
+The following attributes will always be ignored when provided in the request body:
+
+- `id`
```

Comment:
In these scenarios, it's best to list what can be overwritten (whitelist) than what can't (blacklist).
Let's say tomorrow you add a new field to the dashboard model, but forget to ensure how it behaves on clone. You are effectively telling users it works as they would expect (ie. you can provide an overwrite value)

Reply:
Agreed 👍